### PR TITLE
add readme for satbias converter

### PR DESCRIPTION
**Pull request by shlyaeva**
## Description

Adds documentation for satbias2ioda converter in the `README.md`.

### Issue(s) addressed

- fixes https://github.com/jcsda-internal/ioda-converters/issues/417

### CODE REVIEW

File: src/satbias/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+GSI Satbias Converter is a tool that converts GSI files with bias correction coefficients (satbias)
+to the NetCDF files written using IODA ObsGroup capability. One NetCDF file is created for each
+sensor.
+
+## Configuration
+
+The configuration file specifies which sensors need to be processed from the input GSI satbias files,
+the output file names and the names of the predictors for the new file.
+An example of a configuration file can be found in `ioda_converters/test/testinput/satbias_converter.yaml`:
```
Comment (aerorahul) [inline src/satbias/README.md:11]: can you turn this into a hyperlink:
[`test/testinput/satbias_converter.yaml`](https://github.com/JCSDA-internal/ioda-converters/blob/develop/test/testinput/satbias_converter.yaml)

File: src/satbias/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+output:
+- sensor: amsua_n15             # name of sensor as written in GSI satbias_in file
+  output file: satbias_amsua_n15.nc4  # output file name
+  predictors: *default_preds          # list of predictors (use default predictors for amsu*/mhs/atms)
+- sensor: amsua_n19
+  output file: satbias_amsua_n19.nc4
+  predictors: *default_preds
+```
+
+According to [GSI code (as of Feb 19 2021)](https://github.com/NOAA-EMC/GSI/blob/a9447258bb911d73ded0331141d0bf56abdfd74d/src/gsi/setuprad.f90#L992)),
```
Comment (aerorahul) [inline src/satbias/README.md:24]: extra trailing `)`

**Review by aerorahul — APPROVED** [review]
looks good.  a couple of minor suggestions for your consideration.

File: src/satbias/README.md (hunk 1 above)
Comment (shlyaeva) [reply src/satbias/README.md]: Done, thank you!

File: src/satbias/README.md (hunk 2 above)
Comment (shlyaeva) [reply src/satbias/README.md]: thank you for catching this, removed now.

**Review by junmeiban — APPROVED** [review]

File: src/satbias/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# GSI Satbias Converter
+
+GSI Satbias Converter is a tool that converts GSI files with bias correction coefficients (satbias)
```
Comment (fabiolrdiniz) [inline src/satbias/README.md:3]:
```suggestion
GSI Satbias Converter is a tool that converts GSI files with satellite bias correction coefficients (satbias)
```

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @shlyaeva! Just left a suggestion to explicitly say that this file is related to the satellite bias correction component.

**Comment by shlyaeva** [comment]
Thank you @fabiolrdiniz; I've added your suggested change.

**Review by srherbener — APPROVED** [review]
Thanks for adding this documentation!

**Pull request merged**